torchio/data/image.py

```diff
@@ -132,3 +138,23 @@ def load(self, check_nans: bool = True) -> Tuple[torch.Tensor, np.ndarray]:
         if check_nans and torch.isnan(tensor).any():
             warnings.warn(f'NaNs found in file "{self.path}"')
         return tensor, affine
+
+    def is_2d(self) -> bool:
+        return self.shape[-3] == 1
+
+    def numpy(self) -> np.ndarray:
+        return self[DATA].numpy()
+
+    def as_sitk(self) -> sitk.Image:
+        return nib_to_sitk(self.data, self.affine)
+
+    def get_center(self, lps: bool = False) -> TypeTripletFloat:
```

Comment:
may be a even more explicit name : 
get_matrix_center  or get_matrix_center_in_mm

Reply:
Maybe I can write a longer docstring?